loadfile/loadfile_test.go

```diff
@@ -72,7 +72,9 @@ func Test_LoadContext(t *testing.T) {
 	err = fc.LoadContext()
 	assert.Error(t, err)
 	assert.Contains(t, err.Error(), errFileRead)
-	assert.Nil(t, fc.ContentByKey(dirpath))
+	content, err := fc.ContentByKey(dirpath)
+	assert.NoError(t, err)
+	assert.Nil(t, content)
```

Comment:
Just to confirm...in this case, we get _no_ error fetching the content and yet the pointer to that content is _`nil`_?  That seems...somewhere between unsatisfying and outright contradictory.  (If there is no error and no content, shouldn't the value returned be _empty_, not `nil`?...or, is this the thing where an empty slice is `nil`??)

Ditto line 90.